apps/src/templates/projects/LibraryTable.jsx

```diff
@@ -188,16 +193,53 @@ class LibraryTable extends React.Component {
             }
           }
         }
+      },
+      {
+        property: 'actions',
+        header: {
+          label: i18n.quickActions(),
+          props: {
+            style: {
+              ...tableLayoutStyles.headerCell,
+              ...tableLayoutStyles.unsortableHeader
+            }
+          }
+        },
+        cell: {
+          formatters: [this.actionsFormatter],
+          props: {
+            style: {
+              ...tableLayoutStyles.cell,
+              ...styles.cellName,
+              ...styles.centeredCell
+            }
+          }
+        }
       }
     ];
   };
 
+  actionsFormatter = (_, {rowData}) => {
+    return (
+      <Button
+        text={i18n.unpublish()}
+        color={Button.ButtonColor.orange}
+        onClick={() => this.props.unpublishProjectLibrary(rowData.channel)}
+      />
+    );
+  };
+
   render() {
     if (!this.props.personalProjectsList) {
       // Projects haven't loaded from server yet, so display nothing.
       return null;
     }
 
+    const libraries = _.filter(
```

Comment:
Rather than using lodash, you should be able to just do this to keep things clearer:
`const libraries = this.props.personalProjectsList.filter(p => p.libraryName)`